src/App.tsx

```diff
@@ -35,7 +35,10 @@ const Alerts = lazy(() => import('screens/Learn/Alerts/Alerts'));
 const DataApi = lazy(() => import('screens/DataApi/DataApi'));
 const Terms = lazy(() => import('screens/Terms/Terms'));
 const Privacy = lazy(() => import('screens/Terms/Privacy'));
-const Donate = lazy(() => import('screens/Donate/Donate'));
+const Donate = lazy(() => import('screens/Donate/GiveButterDonate'));
```

Comment:
maybe `GiveButterDonate` instead of `Donate`, since there are now multiple variations?

Reply:
It is `GiveButterDonate`?

Reply:
`const GiveButterDonate = lazy(() => import('screens/Donate/GiveButterDonate'))`